lib/src/components/NxTextLink/__tests__/NxTextLink.test.tsx

```diff
@@ -5,114 +5,106 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
-import { getShallowComponent } from '../../../__testutils__/enzymeUtils';
+import { rtlRender, rtlRenderElement } from '../../../__testutils__/rtlUtils';
 
 import NxTextLink from '../NxTextLink';
-import NxFontAwesomeIcon from '../../NxFontAwesomeIcon/NxFontAwesomeIcon';
-import { IconProp } from '@fortawesome/fontawesome-svg-core';
 
 describe('NxTextLink', function() {
-  const getShallow = getShallowComponent(NxTextLink, {});
+  const quickRender = rtlRender(NxTextLink, {});
+  const renderEl = rtlRenderElement(NxTextLink, {});
 
   it('renders an <a> with the nx-text-link class', function() {
-    expect(getShallow()).toMatchSelector('a.nx-text-link');
+    const el = renderEl()!;
+
+    expect(el.tagName).toBe('A');
+    expect(el).toHaveClass('nx-text-link');
   });
 
   it('adds additional specified classes', function() {
-    const component = getShallow({ className: 'foo' });
+    const el = renderEl({ className: 'foo' });
 
-    expect(component).toHaveClassName('foo');
-    expect(component).toHaveClassName('nx-text-link');
+    expect(el).toHaveClass('foo');
+    expect(el).toHaveClass('nx-text-link');
   });
 
   it('adds additional specified props', function() {
-    const component = getShallow({ id: 'foo', lang: 'en_US' });
+    const el = renderEl({ id: 'foo', lang: 'en_US' });
 
-    expect(component).toHaveProp('id', 'foo');
-    expect(component).toHaveProp('lang', 'en_US');
+    expect(el).toHaveAttribute('id', 'foo');
+    expect(el).toHaveAttribute('lang', 'en_US');
   });
 
   it('adds the nx-text-link--external class if external is true', function() {
```

Comment:
Implementation detail.  The relevant external behavior is the presence of the external link icon, which is tested by visual tests

Reply:
> Implementation detail. The relevant external behavior is the presence of the external link icon, which is tested by visual tests

Removed here: 4c42970c97bdf179a2b81e369280dce5396bc4de